beta/src/stores/app.js

```diff
@@ -252,7 +252,7 @@ function app () {
 
       if (redirect) {
         emitter.emit('redirect', {
-          dest: '/login',
+          dest: '/api/v3/user/connect/resonate',
```

Comment:
You need the full url instead of the relative one so the redirect function uses window.location after defined time out.